src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ExternalMovementService.kt

```diff
@@ -217,7 +217,7 @@ class ExternalMovementService(
     val now = LocalDateTime.now()
     return movementTime?.let {
       return if (movementTime.isAfter(now)) {
-        failure(BadRequestException("Transfer cannot be done in the future"))
+        failure(BadRequestException("Movement cannot be done in the future"))
```

Comment:
Decided to bite the bullet and make this error message more generic - [can't find any other projects](https://github.com/search?q=org%3Aministryofjustice+%22cannot+be+done+in+the+future%22&type=code) relying on the exact text so should be good.

Reply:
100%